src/rdiff_backup/Globals.py

```diff
@@ -21,8 +21,10 @@
 import re
 import os
 
+from . import Version
```

Comment:
Once your personal / packaging requirements are clearer we can add here a try/except and do some magic in case the Version.py file doesn't exist.